.vscode/settings.json

```diff
@@ -1,6 +1,7 @@
 {
     "go.formatFlags": ["-s"],
     "go.lintTool": "golangci-lint",
+    "go.lintFlags": ["--fast", "--allow-parallel-runners"],
```

Comment:
On my (newly installed) VSCode w/ Copilot, I was having horrendous performance with basic code inspection and autocomplete w/o upgrading my `golangci-lint` and including these settings.